src/lib/server-utilities.js

```diff
@@ -0,0 +1,9 @@
+import fs from 'fs';
+import crypto from 'crypto';
+
+export function md5Hash(file) {
+  const hashInstance = crypto.createHash('md5');
+  // readFileSync in the syncronous version of readFile
+  const fileContents = fs.readFileSync(file, 'utf8');
+  return hashInstance.update(fileContents).digest('hex');
+}
```

Comment:
Yep that all makes sense to me :). I'll quickly do a change of file names to fit that.